packages/cli/native-node-modules-plugin.ts

```diff
@@ -0,0 +1,39 @@
+import { Plugin } from 'esbuild'
+import path from 'path'
+
+// Source: https://github.com/evanw/esbuild/issues/1051#issuecomment-806325487
+export const nativeNodeModulesPlugin: Plugin = {
+	name: 'native-node-modules',
+	setup(build) {
+		// If a ".node" file is imported within a module in the "file" namespace, resolve
+		// it to an absolute path and put it into the "node-file" virtual namespace.
+		build.onResolve({ filter: /\.node$/, namespace: 'file' }, (args) => ({
+			path: path.join(args.resolveDir, args.path),
```

Comment:
The [original plugin](https://github.com/evanw/esbuild/issues/1051%23issuecomment-806325487) is CommonJS JavaScript. I tried to use `import.meta.resolve()` here but couldn't get it to work. `path.join()` seems to work fine but I'm not 100% sure if this is correct.

Reply:
seems fine to me